src/components/Blinds/Blinds.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import blindsOpen from "../../../html-vzor/images/blinds-open.svg";
+import blindsClosed from "../../../html-vzor/images/blinds-closed.svg";
+import "./Blinds.css";
+
+const Blinds = ({ initialState }) => {
+  console.log(initialState);
+  const [open, setOpen] = useState(Boolean(initialState === "open"));
+
+  return (
+    <div className="blinds">
+      <img src={open ? blindsOpen : blindsClosed} alt="" />
+      <p> Žaluzie </p>
+      <div className="button-container">
+        <button id={open ? "open" : ""} onClick={() => setOpen(true)}>
```

Comment:
ID atribut se v reactu nepoužívá - respektive tahle ne. Jde o to že html id má být na stránce unikátní. A když uděláš tohle, tak komponenta Blinds už nejde na stránce použít víc než jednou. Tj kdybys chtěla mermomocí id použít, např když máš <label for=xx, tak se to dělá takhle:
```
const Component = () => {
   const [id] = useState(uuid());
   
   <label for={id}>....
   <input id={id} ...
}
```
Tohle zajistí že po celý životní cyklus komponenty má jedno stabilní ale unikátní ID.
- ta funkce je z například knihovny https://www.npmjs.com/package/uuid
- nepoužívá se useMemo protože to se může react rozhodnout znovu pustit
- nepoužívá se useState+useEffec (`useEffect(() => setId(uuid()), [])`) protože tam by se komponenta překrelila jednou navíc
- šel by použít useRef
```
const Component = () => {
   const idRef = useRef(uuid());
   
   <label for={idRef.current}>....
   <input id={idRef.current} ...
}
```
ale tam se mi nelíbí ten .current, je to moc psaní :D